### PR TITLE
link to docker.com for local Docker install; fix anchor link

### PR DESCRIPTION
**Pull request by mgannholm**

### CODE REVIEW

File: docs/getting-started-guides/docker.md [hunk 1]
```diff
@@ -30,12 +30,12 @@ Documentation for other releases can be found at
 <!-- END STRIP_FOR_RELEASE -->
 
 <!-- END MUNGE: UNVERSIONED_WARNING -->
-Running Kubernetes locally via Docker
```
Comment (krousey) [inline docs/getting-started-guides/docker.md]: @mgannholm Are these capitalization changes intentional?

Comment (mgannholm) [inline docs/getting-started-guides/docker.md]: the capitalization changes were made by someone else -- github said my changes (which were non-conflicting) would merge fine...

Comment (krousey) [inline docs/getting-started-guides/docker.md]: Unfortunately, the diff here is probably what will be committed. If you don't mind, could you change them back?

**Comment by wojtek-t** [comment]
@mgannholm - can you please rebase or close this PR?

**Pull request closed without merging**